File: app/src/main/java/world/hit/com/hitman/MainActivity.java
package world.hit.com.hitman;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.SeekBar;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    private SeekBar _seekBar;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        _seekBar = (SeekBar) findViewById(R.id.seekBar);

        setupUI();
    }

    private void setupUI(){
        _seekBar.setOnSeekBarChangeListener(new SeekBar.OnSeekBarChangeListener() {
            @Override
            public void onProgressChanged(SeekBar seekBar, int progress, boolean fromUser) {
            }

            @Override
            public void onStartTrackingTouch(SeekBar seekBar) {

            }

            @Override
            public void onStopTrackingTouch(SeekBar seekBar) {
                showToast(String.valueOf(seekBar.getProgress()));
            }
        });
    }

    private void showToast(String msg){
        Toast.makeText(this,"Cursor at : "+msg,Toast.LENGTH_LONG).show();
    }
}

File: app/src/main/java/world/hit/com/hitman/MainActivity.kt
package world.hit.com.hitman

import android.support.v7.app.AppCompatActivity
import android.os.Bundle
import android.widget.SeekBar
import android.widget.Toast

class MainActivity : AppCompatActivity() {
    private var _seekBar: SeekBar? = null

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        _seekBar = findViewById(R.id.seekBar) as SeekBar

        setupUI()
    }

    private fun setupUI() {
        _seekBar!!.setOnSeekBarChangeListener(object : SeekBar.OnSeekBarChangeListener {
            override fun onProgressChanged(seekBar: SeekBar, progress: Int, fromUser: Boolean) {}

            override fun onStartTrackingTouch(seekBar: SeekBar) {

            }

            override fun onStopTrackingTouch(seekBar: SeekBar) {
                showToast(seekBar.progress.toString())
            }
        })
    }

    private fun showToast(msg: String) {
        Toast.makeText(this, "Cursor at : " + msg, Toast.LENGTH_LONG).show()
    }
}
